cmd/bucket-notification-utils_test.go

```diff
@@ -337,11 +332,6 @@ func TestUnmarshalSQSARN(t *testing.T) {
 			queueARN: "",
 			Type:     "",
 		},
-		// Invalid region 'us-west-1' in queue arn.
-		{
-			queueARN: "arn:minio:sqs:us-west-1:1:redis",
```

Comment:
Are you sure that there is a test case which covers are scenario where client sets a region which is different than the one set on the server?. I am not seeing it. @krishnasrinivas